Integrations-NotTested/integrations-proofpoint_threat_response.yml

```diff
@@ -0,0 +1,365 @@
+            json=member
+        )
+
+        if res.status_code < 200 or res.status_code >= 300:
+            return_error('Add to list failed. URL: {}, Request Body: {}, StatusCode: {}, Response: {}'.format(fullurl, json.dumps(member), res.status_code, res.content))
+
+        return res.json()
+
+    def add_to_list_command():
+        ''' Adds given members to the given list ID in Threat Response '''
+        list_id = demisto.args().get('list-id')
+        members = argToList(demisto.args().get('member'))
+        comment = demisto.args().get('comment')
+        expiration = demisto.args().get('expiration')
+
+        message = ''
+        for member in members:
+            add_to_list(list_id, member, comment, expiration)
+            message += '{} added successfully to {}\n'.format(member, list_id)
+
+        demisto.results(message)
+
+
+    def block_ip_command():
+        ''' Adds given IPs to the relevant blacklist in Threat Response '''
+        list_id = demisto.params().get('blacklist_ip')
+        members = argToList(demisto.args().get('ip'))
+        expiration = demisto.args().get('expiration')
+
+        message = ''
+        for member in members:
+            add_to_list(list_id, member, None, expiration)
+            message += '{} added successfully to block_ip list\n'.format(member)
+
+        demisto.results(message)
+
+
+    def block_domain_command():
+        ''' Adds given domains to the relevant blacklist in Threat Response '''
+        list_id = demisto.params().get('blacklist_domain')
+        members = argToList(demisto.args().get('domain'))
+        expiration = demisto.args().get('expiration')
+
+        message = ''
+        for member in members:
+            add_to_list(list_id, member, None, expiration)
+            message += '{} added successfully to block_ip list\n'.format(member)
+
+        demisto.results(message)
+
+
+    def block_url_command():
+        ''' Adds given URLs to the relevant blacklist in Threat Response '''
+        list_id = demisto.params().get('blacklist_url')
+        members = argToList(demisto.args().get('url'))
+        expiration = demisto.args().get('expiration')
+
+        message = ''
+        for member in members:
+            add_to_list(list_id, member, None, expiration)
+            message += '{} added successfully to block_url list\n'.format(member)
+
+        demisto.results(message)
+
+
+    def block_hash_command():
+        ''' Adds given hashes to the relevant blacklist in Threat Response '''
+        list_id = demisto.params().get('blacklist_hash')
+        members = argToList(demisto.args().get('hash'))
+        expiration = demisto.args().get('expiration')
+
+        message = ''
+        for member in members:
+            add_to_list(list_id, member, None, expiration)
+            message += '{} added successfully to block_hash list\n'.format(member)
+
+        demisto.results(message)
+
+
+    def search_members(list_id, member_filter):
```

Comment:
What entity do we refer to as member and lists here?

Reply:
list_id refers to a specific blacklist in the product, each list contains different values(a list for urls, another for hashes, etc)
member - a member of the list, can be anything, depends on the list.

Reply:
@bakatzir change member terminology to indicator

Reply:
Done